platform/ios/app/MBXViewController.m

```diff
@@ -929,7 +929,7 @@ - (void)styleBuildingExtrusions
 
         MGLFillExtrusionStyleLayer* layer = [[MGLFillExtrusionStyleLayer alloc] initWithIdentifier:@"extrudedBuildings" source:source];
         layer.sourceLayerIdentifier = @"building";
-        layer.predicate = [NSPredicate predicateWithFormat:@"extrude == 'true' AND height > 0"];
+        layer.predicate = [NSPredicate predicateWithFormat:@"extrude == 'true' AND height != 0"];
```

Comment:
Good catch! Not sure if it’s possible for the height to be negative; I think the Streets source might already exclude those underground buildings (such as subway stations).

The root cause is that `height` needs to be cast to a number before it can be used in an inequality comparison, so `CAST(height, 'NSNumber') > 0` should do the trick. This is a change from previous releases of the map SDK, in which key paths were cast automatically as needed.